iceoryx_hoofs/test/moduletests/test_relative_pointer.cpp

```diff
@@ -528,23 +435,23 @@ TEST_F(base_relative_ptr_test, memoryRemapping)
     }
 }
 
-TEST_F(base_relative_ptr_test, MemoryReMapping_SharedMemory)
+TYPED_TEST(base_relative_ptr_test, MemoryReMapping_SharedMemory)
 {
-    MemMap memMapWriter(this->m_fileDescriptor);
-    MemMap memMapReader(this->m_fileDescriptor);
+    void* memoryWriter = this->memoryPartition[0];
+    void* memoryReader = this->memoryPartition[0];
```

Comment:
This test is now kind of pointless. Previously it was tested that the same memory was mapped to different address spaces and the memory was accessed by different addresses. Now it's the same address for the reader and the writer and it doesn't make sense anymore.